vms/evm/subscriber.go

```diff
@@ -118,23 +217,13 @@ func (s *subscriber) Subscribe() error {
 func (s *subscriber) dialAndSubscribe() error {
 	// Dial the configured destination chain endpoint
 	// This needs to be a websocket
-	ethClient, err := ethclient.Dial(s.nodeURL)
+	ethClient, err := ethclient.Dial(s.nodeWSURL)
```

Comment:
- Why do we use websocket for the dial and subscribe, which does `SubscribeFilterLogs`, vs rpc url which does `FilterLogs`? 
- Is the comment meant to be source chain endpoint?

Reply:
`FilterLogs` calls the RPC method `eth_getLogs` which grabs all of the historical logs that match a particular query, while `SubscribeFilterLogs` calls the eth_subscribe method to establish a websocket connection to transmit real time logs as they are finalized on chain.

But yes, that comment should read `source chain`